Common/src/main/java/me/egg82/tfaplus/utils/ConfigUtil.java

```diff
@@ -0,0 +1,127 @@
+package me.egg82.tfaplus.utils;
+
+import com.authy.AuthyApiClient;
+import com.authy.api.Tokens;
+import com.google.common.collect.ImmutableSet;
+import com.rabbitmq.client.ConnectionFactory;
+import me.egg82.tfaplus.TFAAPI;
+import me.egg82.tfaplus.core.FreezeConfigContainer;
+import me.egg82.tfaplus.enums.SQLType;
+import me.egg82.tfaplus.extended.CachedConfigValues;
+import me.egg82.tfaplus.extended.Configuration;
+import ninja.egg82.service.ServiceLocator;
+import ninja.egg82.service.ServiceNotFoundException;
+import ninja.egg82.sql.SQL;
+import ninja.leaping.configurate.ConfigurationNode;
+import redis.clients.jedis.JedisPool;
+
+import javax.annotation.Nullable;
+import java.util.Optional;
+
+public class ConfigUtil {
+    /**
+     * Grabs the config instance from ServiceLocator
+     * @return instance of the Configuration class, may return null
+     */
+    public static @Nullable Configuration getConfig()
+    {
+        Configuration config;
+
+        try {
+            config = ServiceLocator.get(Configuration.class);
+            return config;
+        } catch (IllegalAccessException | InstantiationException | ServiceNotFoundException ex) {
+            TFAAPI.getLogger().error(ex.getMessage(), ex);
+        }
+
+        return null;
+    }
+
+    /**
+     * Grabs the cached config instance from ServiceLocator
+     * @return instance of the CachedConfigValues class, may return null
+     */
+    public static @Nullable CachedConfigValues getCachedConfig()
+    {
+        CachedConfigValues cachedConfig;
+
+        try {
+            cachedConfig = ServiceLocator.get(CachedConfigValues.class);
+            return cachedConfig;
+        } catch (IllegalAccessException | InstantiationException | ServiceNotFoundException ex) {
+            TFAAPI.getLogger().error(ex.getMessage(), ex);
+        }
+
+        return null;
+    }
+
+    public static @Nullable SQL getSQL()
+    {
+        return getCachedConfig().getSQL();
+    }
+
+    public static Tokens getTokens()
```

Comment:
Probably would name this `getAuthyTokens` so I don't confuse myself later when adding more 2FA providers.